frontend/src/metabase/dashboard/containers/CreateDashboardForm.tsx

```diff
@@ -108,18 +88,25 @@ function CreateDashboardForm({
       onSubmit={handleCreate}
     >
       {() => (
-        <Form>
-          <FormInput
+        <Form as={Stack} spacing={0}>
+          <FormTextInput
+            labelProps={{ mb: "xs", fw: "800" }}
```

Comment:
nit: use numeric values for weights

```suggestion
            labelProps={{ mb: "xs", fw: 800 }}
```